doc/model/dplr.md

```diff
@@ -0,0 +1,171 @@
+# Deep potential long-range (DPLR)
+
+:warning: **The interfaces of DPLR are not stable and subject to change**
```

Comment:
`warning` does not work. https://deepmodeling--1458.org.readthedocs.build/projects/deepmd/en/1458/model/dplr.html#deep-potential-long-range-dplr

Reply:
It should be
```rst
.. warning::
    Something
```